### DISCUSSION TITLE
There is no response after the result is submitted

### DISCUSSION THREAD

**Discussion by gaozhanfire** [pykt-team/pykt-toolkit#78] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34774408/205432597-144a01d4-0e67-4031-895f-2e07d2718fe4.png)
I submitted it several times before, and then I got the output quickly. But now I have been waiting for a long time, and there is no output.

**Comment by meta-tabchen** [comment, marked as answer]
Hi, this might be a problem with the platform, it's possible that too many people are submitting causing the delay of computing resources. We checked the background, now your results should be out, if you still can't see it, you can leave a message, and we will deal with it.

pyKT team
